src/styles/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
```

Comment:
not sure if it's necessary, but you could assign colours to a variables and using them that way. but again, there's no issue, just a suggestion

Reply:
agree, better use variables for commonly used values, such as colors, please do it in this task